### PR TITLE
Add ProjectController@toggle_watch spec to RSpec

### PR DESCRIPTION
**Pull request by mdeniz**

### CODE REVIEW

**Comment by mdeniz** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;wY1WY48T&#x2F;6-s-write-test-for-app-controllers-webui-project-controller-rb-toggle-watch"> [s] Write test for app&#x2F;controllers&#x2F;webui&#x2F;project_controller.rb#toggle_watch</a>

File: src/api/spec/controllers/webui/project_controller_spec.rb [hunk 1]
```diff
@@ -659,4 +659,33 @@
...
+
+  describe 'GET #toggle_watch' do
+    before do
+      login user
+    end
+
+    it "with a project already whatched" do
+      User.current.add_watched_project(user.home_project_name)
+      get :toggle_watch, project: user.home_project
+      expect(User.current.watched_project_names).not_to include(user.home_project_name)
```
Comment (bgeuken) [inline src/api/spec/controllers/webui/project_controller_spec.rb]: ``` expect(user.watched_project_names)``` ;-)

File: src/api/spec/controllers/webui/project_controller_spec.rb [hunk 2]
```diff
@@ -659,4 +659,33 @@
...
+    end
+
+    it "with a project not whatched" do
+      User.current.remove_watched_project(user.home_project_name)
```
Comment (bgeuken) [inline src/api/spec/controllers/webui/project_controller_spec.rb]: Don't think you need that here.

File: src/api/spec/controllers/webui/project_controller_spec.rb [hunk 3]
```diff
@@ -659,4 +659,33 @@
...
     end
   end
+
+  describe 'GET #toggle_watch' do
+    before do
+      login user
+    end
+
+    it "with a project already whatched" do
+      User.current.add_watched_project(user.home_project_name)
```
Comment (bgeuken) [inline src/api/spec/controllers/webui/project_controller_spec.rb]: Could you add a factory instead? Please:-)

Comment (mdeniz) [inline src/api/spec/controllers/webui/project_controller_spec.rb]: There is already a WatchedProject factory, I will use it instead

File: src/api/spec/controllers/webui/project_controller_spec.rb (hunk 2 above)
Comment (mdeniz) [inline src/api/spec/controllers/webui/project_controller_spec.rb]: Yes you are right... :+1:

**Comment by bgeuken** [comment]
LGTM

**Pull request merged**